src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -35,7 +35,7 @@ class ProfileView extends React.Component {
     if (this.props.isVet360AvailableForUser) {
       this.props.fetchTransactions();
     } else {
-      // this.props.initializeUserToVet360()
+      this.props.initializeUserToVet360();
     }
```

Comment:
I think we'll also need to check `this.props.isVet360AvailableForUser` during render and if false return a `<TransactionPending/>` that we can use for messaging and to poll for updates. Without that, I think the application will proceed as if the user was initialized from the start and we might see some weird behavior.

Reply:
We do have messaging in Contact information that renders `<MissingVet360IDError/>` if the `isVet360AvailableForUser` is false; is there some other way you are thinking of doing messaging or handling actions in the non-initialized state?

Reply:
Some messaging about the initialization happening might be nice, but my main concern is that since we're not polling for updates on the initialization transaction, the application won't "unlock" once the transaction completes and the user will think there's some indefinite error on the Profile

Reply:
Does that make sense?

Reply:
Ah you know what I am missing the polling functionality in this PR. I will move the issue back to in-progress and get the polling/auto-refresh in here